test/fuzz/interprets.ml

```diff
@@ -73,6 +73,36 @@ module Owi_optimized : INTERPRET = struct
   let name = "owi+optimize"
 end
 
+module Owi_symbolic : INTERPRET = struct
+  type t = Text.modul
+
+  let of_symbolic = Fun.id
+
+  let run modul =
+    match Compile.until_simplify ~unsafe:false modul with
+    | Error e -> failwith e
+    | Ok simplified -> (
+      match Typecheck.modul simplified with
+      | Error e -> failwith e
+      | Ok () -> (
+        match Link.modul Link.empty_state ~name:None simplified with
+        | Error e -> failwith e
+        | Ok (regular, link_state) ->
+          let regular = Symbolic.convert_module_to_run_minimalist regular in
+          timeout_call_run (fun () ->
+              let c = Interpret.SymbolicM.modul link_state.envs regular in
+              let init_thread : Thread.t = Thread.create () in
+              let res, _ =
+                Symbolic_choice.Minimalist.run_minimalist c init_thread
+              in
+              match res with
+              | Ok res -> res
+              | Trap _t -> Result.error "symbolic trap"
+              | Assert_fail -> Result.error "symbolic assert_fail" ) ) )
```

Comment:
```ocaml
let run modul = 
  open Syntax in
  let* simplified = ... in
  let* () = ... in
  let* (regular, link_state) = ... in
  ...
  
let run modul =
  match run modul with
  | Error e -> failwith e
  | Ok v -> v
```

Reply:
(and it looks like the second function is not even necessary)

Reply:
sorry, I don't understand :(

Reply:
If you open the `Owi.Syntax` module, you can rewrite all your code with a `let*` instead of having a lot of (hard to read) nested `match`.

Reply:
> If you open the `Owi.Syntax` module, you can rewrite all your code with a `let*` instead of having a lot of (hard to read) nested `match`.

And i used the same approach in `Owi_optimized` and `Owi_unoptimized` modules

Reply:
> (and it looks like the second function is not even necessary)

Is this part of the review ok now?

Reply:
Yes. :)